code/game/objects/items/devices/flashlight.dm

```diff
@@ -458,7 +460,7 @@
 	var/trash_type = /obj/item/trash/flare
 	/// If the light source can be extinguished
 	var/can_be_extinguished = FALSE
-	custom_materials = list(/datum/material/plastic= SMALL_MATERIAL_AMOUNT * 0.5)
+	custom_materials = list(/datum/material/plastic = SHEET_MATERIAL_AMOUNT)
```

Comment:
This has the consequence of having to go through all the recipes that require flares, and their resulting items as well, to make sure it's consistent.